packages/react-instantsearch/src/connectors/connectConfigure.js

```diff
@@ -1,7 +1,10 @@
 import createConnector from '../core/createConnector.js';
-import {omit, isEmpty, difference, keys} from 'lodash';
+import {omit, difference, keys} from 'lodash';
```

Comment:
same as earlier, we can also use https://github.com/lodash/babel-plugin-lodash or https://github.com/lodash/lodash-webpack-plugin